blocks/extensions/plot.py

```diff
@@ -136,14 +130,103 @@ def preexec_fn():
             logger.info('Plotting server PID: {}'.format(self.sub.pid))
         else:
             self.sub = None
-        output_server(self.document, url=self.server_url)
+
+        self.session = Session(
+            root_url=self.server_url,
```

Comment:
Judging from the errors on Travis, it's this line that is causing problems. If it's not for you, that might be because of different Bokeh versions?